polyphemus/lib/client/jsx/etl/run-pane.tsx

```diff
@@ -247,18 +244,18 @@ const RunPane = ({
           </Select>
         </Grid>
         {runState == RUN_INTERVAL && (
-          <Grid item>
+          <Grid item xs={4}>
             {' '}
             <TextField
               size='small'
               InputProps={{
                 endAdornment: (
-                  <InputAdornment position='end'>seconds</InputAdornment>
+                  <InputAdornment position='end'>{'seconds (300 minimum)'}</InputAdornment>
                 )
               }}
               value={runIntervalTime}
               onChange={(e) => {
-                setRunIntervalTime(parseInt(e.target.value as string));
+                setRunIntervalTime(Math.max(300,parseInt(e.target.value as string)));
               }}
```

Comment:
Could put the restriction on the input instead.
```suggestion
                setRunIntervalTime(parseInt(e.target.value as string));
              }}
              inputProps={{
                min: 300,
              }}
```

Reply:
Just tested this and what you had before is better. `min` in inputProps only enforces if you press the up/down buttons or arrow keys to change the value.